action/protocol/execution/protocol.go

```diff
@@ -21,7 +21,8 @@ import (
 
 const (
 	// ExecutionSizeLimit is the maximum size of execution allowed
-	ExecutionSizeLimit = 32 * 1024
+	ExecutionSizeLimit    = 64 * 1024
+	_executionSizeLimit32 = 32 * 1024
```

Comment:
should be 48*1024=49152? according to https://eips.ethereum.org/EIPS/eip-3860
also, this const is only used in its own package, so could also be internal
`_executionSizeLimit48kb = 48 * 1024`

Reply:
okay, 48K is enough